src/Typography/Typography.story.tsx

```diff
@@ -1,7 +1,8 @@
 import { storiesOf } from '@storybook/react';
 import faker from 'faker';
 import React, { Fragment } from 'react';
-import Typography from '.';
+
+import Typography from './Typography';
```

Comment:
Why does this need to change if we still have default exports?

Reply:
It follows the convention of not using the index file internally. In a nutshell, when interfacing with a module externally, you _only ever use its index_. On the flip side, when interfacing internally, you _never use the index_. It doesn't matter as much here, but the practice prevents circular dependencies from forming.

Reply:
Is this required because of how we're doing global imports, or is this a purely stylistic change?

Reply:
It's a best-practices for TS/node.js thing. Basically, there's no drawback, and it can avoid circular dependency issues, so it's a good hold-fast, especially since the project is young so it doesn't require much rework.

Reply:
Personally I think it would be much simpler to just remove default exports

Reply:
I'm curious as to why this alone makes you want to remove defaults -- at first glance, it doesn't seem to have anything to do with named vs. not-named; I would still lobby for `./Foo` vs. `.`.

I think default exports are more semantic, tbh.

Reply:
I'm not sure I understand exactly what sort of situation this helps prevent, but I'm fine with it if we can lint it.